docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,233 @@
+
+1. From the head of the release branch (``tickets/RFC-N`` or ``tickets/LCR-N``), add a commit that:
+
+	- Updates the semantic version of the document to the one designated by the project librarian.
+	- Updates the change history table to reflect the new version.
+	- Fixes the document's date to the date of baselining.
+	- Removes the "draft" watermark (for example, by removing ``draft`` from the document class's arguments).
+
+2. Create a new DocuShare tag (``docushare-vN``) and upload the document to DocuShare.
+
+3. Once the project librarian has set that DocuShare version as the preferred version, create a semantic version tag (formatted ``v<major>.<minor>``) at the same commit as the corresponding DocuShare tag.
+
+4. Create a new ticket branch from the **head** of the ``master`` branch.
+   Cherry pick amendment commits from the release branch onto that ticket branch.
+   Also include a commit that updates the document change history table.
+   Merge that ticket branch to ``master``.
+
+**Notes:**
+
+- The release branch is never merged back to ``master``.
+  Amendments get back to ``master`` through cherry picking commits from the release branch.
+- Development is allowed to happen on the ``master`` branch while CCB review is simultaneously happening on a release branch.
+  This means that the DM release manager is responsible for properly addressing conflicts while cherry picking amendments back to ``master``.
+
+.. figure:: ccd-release.svg
+   :name: fig-release
+   :align: center
+   :target: ccd-release.svg
+   :alt: Git commit tree showing a document release.
+
+   Example Git tree showing how a document is released and baselined.
+   The release manager creates a ticket branch (``tickets/DM-6``) with two commits from the current release branch (``tickets/RFC-1``).
+   Commit ``D`` updates the document's history table.
+   Commit ``E`` sets the document's version of removes the draft watermark.
+   Once this ticket branch is merged to the release branch, the release manager uploads it to document (using a ``docushare-v3`` tag) and also tags the document's semantic version (``v1.0``).
+   Finally, the release manager backports the amendment commits (``A``, ``B``, ``C``, and ``D`` --- but not ``E`` where the draft watermark was removed) to the master branch using a ticket branch (``tickets/DM-7``).
+
+.. _ccd-hotfix:
+
+Hotfixing a baselined document
+==============================
+
+It may be necessary to release a minor update to a baselined document (to fix typos, for example).
+This is done by creating a branch based off the prior release branch.
+Hotfixes cannot be made from ``master`` because significant edits may have been merged since the document was baselined.
+
+1. Create a ticket branch from the semantic version tag of the prior release.
+2. Commit fixes to that ticket branch and have the changes peer reviewed.
+3. Create a DocuShare tag (``docushare-vN``) and upload the document to DocuShare.
```

Comment:
standard ordering problem. Maybe we can simplify the repeat of the logic here by saying that this is exactly like a normal CCB submission but the branch point is the prior version tag rather than master?